tests/models/maskformer/test_modeling_maskformer_swin.py

```diff
@@ -146,18 +143,13 @@ def create_and_check_backbone(self, config, pixel_values, labels):
         result = model(pixel_values)
 
         # verify feature maps
-        self.parent.assertEqual(len(result.feature_maps), len(config.out_features))
+        self.parent.assertEqual(len(result.feature_maps), len(config.out_indices))
         self.parent.assertListEqual(list(result.feature_maps[0].shape), [13, 16, 16, 16])
 
         # verify channels
-        self.parent.assertEqual(len(model.channels), len(config.out_features))
+        self.parent.assertEqual(len(model.channels), len(config.out_indices))
         self.parent.assertListEqual(model.channels, [16, 32, 64])
 
-        # verify ValueError
-        with self.parent.assertRaises(ValueError):
-            config.out_features = ["stem"]
-            model = MaskFormerSwinBackbone(config=config)
```

Comment:
If there's verification on the `out_features` (making sure one value isn't / can't be selected) this should be updated to check for the corresponding `out_indices` value